dvc/repo/add.py

```diff
@@ -65,7 +83,12 @@ def _find_all_targets(repo, target, recursive):
     if os.path.isdir(target) and recursive:
         return [
             fname
-            for fname in repo.tree.walk_files(target)
+            for fname in Tqdm(
+                repo.tree.walk_files(target),
+                desc="Recursing " + target,
```

Comment:
Not sure I've ever heard or seen `Recursing ` used that way :) Maybe there is a better description? `Collecting files in `? What do you think?

Reply:
Traversing?